### ISSUE TITLE
commands is not a heroku command

### ISSUE THREAD

**Issue by JoeNyland** [JoeNyland/heroku-completion#3]
Seems to have been removed from Heroku.

https://github.com/JoeNyland/heroku-completion/blob/master/heroku-bash-completion.sh#L171 will have to be a hard coded array, unless a dynamic list can be obtained from somewhere 😕

**Comment by JoeNyland** [comment]
Are there any PRs to update when this is complete? 🤔

**Comment by JoeNyland** [comment]
Hmm, getting the following too:

```
This CLI is deprecated. Please reinstall from https://cli.heroku.com
```

**Comment by JoeNyland** [comment]
Fixed in efbcd80936ba632f1bf468d6af3ee812cf00f4d0
